### PR TITLE
FAI-354 - Domain dashboards for DRL 

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/FAI-354

This PR aims to provide a domain dashboard for DRL. In short, the generated dashboard contains the panel that shows the 1-minute average time to evaluate the rules. 

**NOTE**: there is one limitation. If the kogito application contains > 1 RuleUnits, the domain panel will show **all** the rules of all the RuleUnits. This is because in the Drools listener we can't get the information about the corresponding RuleUnit for the metric. 

Kogito-examples update PR: https://github.com/kiegroup/kogito-examples/pull/630

Generated dashboard example: 
Screenshot from 2021-03-26 16-04-01

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/master/kogito-ide-config)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

**Comment by r00ta** [comment]
Jenkins retest this please

File: grafana-api/src/main/java/org/kie/kogito/grafana/GrafanaConfigurationWriter.java [hunk 1]
```diff
@@ -122,10 +98,45 @@ public static String generateDomainSpecificDMNDashboard(String templatePath, Str
...
+        return jgrafana;
+    }
+
+    private static String serialize(JGrafana jgrafana) {
         try {
             return jgrafana.serialize();
         } catch (IOException e) {
-            logger.error(String.format("Could not serialize the grafana dashboard for the endpoint %s", endpoint), e);
+            logger.error("Could not serialize the grafana dashboard");
             throw new RuntimeException("Could not serialize the grafana dashboard.", e);
```
Comment (danielezonca) [inline grafana-api/src/main/java/org/kie/kogito/grafana/GrafanaConfigurationWriter.java:140]:
```suggestion
            throw new UncheckedIOException("Could not serialize the grafana dashboard.", e);
```

**Review by danielezonca — APPROVED** [review]

**Comment by r00ta** [comment]
Hi @jiripetrlik , could you take a look at this one?

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Review by jiripetrlik — APPROVED** [review]
LGTM

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by danielezonca** [comment]
jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
Jenkins retest this please

**Comment by r00ta** [comment]
jenkins retest this please

**Pull request merged**